cmd/cluster-agent/app/servicemapperdigest.go

```diff
@@ -0,0 +1,105 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	log "github.com/cihub/seelog"
+	"github.com/spf13/cobra"
+
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/status"
+)
+
+func init() {
+	ClusterAgentCmd.AddCommand(svcMapperCmd)
+	svcMapperCmd.SetArgs([]string{"caseID"})
+}
+
+// TODO add all registered nodes command
+var svcMapperCmd = &cobra.Command{
```

Comment:
Cobra can provide some examples, what do you think of adding few ?

Reply:
👍